### DISCUSSION TITLE
How to delete a part of a shape?

### DISCUSSION THREAD

**Discussion by playgroundearth** [keiwando/toonsquid#574] [Q&A]
Hello

The eraser tool is not working on shapes. Is this normal?
I draw a shape using the path tool/ shape and need to cut or delete a part to create the shape I need

I was looking to use the eraser.

any suggestions or ideas to help?

thanks

**Comment by kavlor** [comment]
Tap a point which reveals the handles then tap again and it will bring up a menu with delete at the bottom. You can also use masking which can be done as an animation layer(bottom) or drawing layer(right) which allows you to mix Raster and Vector together on one animation layer.

**Comment by playgroundearth** [comment]
Oh I see 

There is no erasing shapes with the eraser
Shapes can be modified. The trimming tool is helping me achieve the cut I was after.

Thanks
